values.yaml

```diff
@@ -1050,6 +1083,13 @@ mongodb:
   arbiter:
     enabled: false
   resources: {}
+  tls:
+    enabled: true
+    replicaset:
+      existingSecrets:
+        - internal-tls
+        - internal-tls
+        - internal-tls
```

Comment:
Why the duplication here?

Reply:
MongoDB helm template requires an `existingSecret` for _each_ mongodb node, I assume with the intention that each node has its own cert - I've however instead created a cert thats valid for all nodes and just use the same one for all nodes - however, the Mongo chart still requires all 3 to be explicitly defined.